content/en/blog/_posts/2020-08-21-Moving-Forward-From-Beta/index.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: blog
+title: "Moving Forward From Beta"
+date: 2020-08-21
+slug: moving-forward-from-beta
+---
+
+**Author**: Tim Bannister, The Scale Factory
+
+In Kubernetes, features follow a defined
+[lifecycle](/docs/reference/command-line-tools-reference/feature-gates/#feature-stages).
+First, as the twinkle of an eye in an interested developer. Maybe, then,
```

Comment:
Could you combine the `First, features appear as the twinkle of an eye in an interested developer. Then, maybe, features are sketched in online discussions, drawn ...`?